### DISCUSSION TITLE
Error de autorización

### DISCUSSION THREAD

**Discussion by joanfco30** [rramosp/ai4eng.v1#116] [04 - GESTIÓN DE DATOS]
Buenos días un cordial saludo, entiendo que debido a una actualización de Google hemos tenido problemas de autentificación para desarrollar el curso.  Saben si hay otra forma de trabajar con el objetivo de recibir las evaluaciones de los retos o se depende 100% de esta actualización. En ese orden, estaré pendiente de cualquier notificación. 

Feliz día.

**Comment by Felipe-RA** [comment]
Saludos joanfco30

Revisa tu correo, recibiste instrucciones de parte del profesor raúl el día de ayer.

Hasta pronto

**Reply by joanfco30** [reply]
Buenos días Felipe un cordial saludo. El correo no lo he recibido aun. Es posible a través de ti enviarlo al siguiente correo joanfco1@gmail.com o jfalvaradm@eafit.edu.co 
Muchas gracias y quedo atento a tus comentarios. 

Feliz día

**Comment by rramosp** [comment]
hola joanfco30, 

tienes razón, habíamos mandado un mensaje a los matriculados este semestre en la udea por los plazos internos de la materia, pero no habíamos notificado en este foro.

acabo de poner el mensaje en la categoría de plataforma.

en resumen, tienes que entrar con password por el momento. resetea tu pwd si lo necesitas o si nunca has entrado con password

espero te funcione, cualquer cosa nos dices
raúl

**Reply by joanfco30** [reply]
Buenos días un cordial saludo. Sigo con el mismo problema.  Entro con password y me sale el siguiente mensaje y también hice resetee mi contraseña.  Por lo que acabo de leer. Es mejor inscribirme al curso desde otro correo ? Esto debido a que al estar inscrito desde el semestre anterior se pierde lo evaluado hasta el momento. Revise las evaluaciones y aun tengo los puntos, lo curioso es que puedo las evaluaciones si las puedo ver. 
Muchas gracias y seguiré atento
![image](https://user-images.githubusercontent.com/60493083/161097024-5020169b-f521-46c5-8ede-6b973c5ac2ad.png)
 
![image](https://user-images.githubusercontent.com/60493083/161095591-8d3b0f0a-534d-4272-937a-31dcc97e24cc.png)
![image](https://user-images.githubusercontent.com/60493083/161095779-534ff109-3387-4789-9b92-5b48c17427eb.png)

**Reply by Felipe-RA** [reply]
Saludos joanfco30

Si estas ingresando desde el campo de user-password?

![image](https://user-images.githubusercontent.com/49454068/161097907-fa0dfa07-48fd-4be9-8740-105fa5234f3f.png)

**Reply by joanfco30** [reply]
Hola Andres buenos días si.  Y aun así no me permite ingresar. 
![image](https://user-images.githubusercontent.com/60493083/161100713-98bc38ca-2dcb-45ae-9345-f2b720953c78.png)

![image](https://user-images.githubusercontent.com/60493083/161100487-186c2a3a-cf25-40a0-a412-c0de72b13703.png)

**Reply by joanfco30** [reply]
Andres ya solucione. Agradezco tu respuesta y tus ayudad. Nuevamente muchas gracias. 

![image](https://user-images.githubusercontent.com/60493083/161102519-1b285352-c690-48c3-b1b8-b543baba1ed5.png)
